### DISCUSSION TITLE
Question re: orswring on PCBWay

### DISCUSSION THREAD

**Discussion by ameenb** [HDR/Gameboy-Camera-Flashcart#7] [Q&A] [answered]
Hi! I attempted to place an order on PCBWay for a cart and I believe I followed all of your specs.

I got a reply from them with the following question I don't know the answer to, can you assist please? Desperate to get my hands on one of these babies. Thank you!

"As shown in picture,for PCB order W465918ASB1,these circles pads lessing soldermask opening,please check if opening for these pads are missing?

Please adjust design file,and send new file to me."
![W465918ASB1](https://user-images.githubusercontent.com/8595602/202454098-55da69f1-86c7-439a-9d41-8dd5832b9e45.png)

**Comment by HDR** [comment, marked as answer]
Tell them to follow the file, a lot of these fabs don't understand that the copper on the back should be covered with mask, you can send them this image as an example.

![image](https://user-images.githubusercontent.com/20230450/202461700-d035887e-0646-49d8-9c97-6c9fee156150.png)
